internal/envoy/route.go

```diff
@@ -41,6 +45,14 @@ func RouteRoute(r *dag.Route, services []*dag.HTTPService) *route.Route_Route {
 		)
 	}
 
+	rlActions := RateLimitActions(r)
```

Comment:
Could you change RateLimitActions to return a []*route.RateLimit, then do something like
```
ra.RateLimits = RateLimitActions(r)
```
If RateLimitActions returns nil, that will slot in just nicely as the 'not set' value for ra.RateLimits.